modules/ezcontent_preview/src/Plugin/views/filter/AccessUnpublishedToken.php

```diff
@@ -22,7 +22,7 @@ class AccessUnpublishedToken extends FilterPluginBase {
    */
   public function init(ViewExecutable $view, DisplayPluginBase $display, array &$options = NULL) {
     parent::init($view, $display, $options);
-    $this->valueTitle = t('Published or Unpublish Access Token');
+    $this->valueTitle = $this->t('Published or Unpublish Access Token');
```

Comment:
This seems to be duplicate changes of PR https://github.com/srijanone/ezcontent_api/pull/22/files, Please see.